tests/output.rs

```diff
@@ -127,7 +127,8 @@ fn strip_trailing_carriage_return(line: &str) -> Cow<'_, str> {
 }
 
 fn normalize_match_path(line: &str) -> Cow<'_, str> {
-    regex!(r#"^[^:]+:"#).replace(line, |captures: &Captures| captures[0].replace('\\', "/"))
+    regex!(r#"^[^:]+[:-]\d+[:-]"#)
```

Comment:
Windows CI was terribly offended that I hadn't updated this regex to now also massage context output lines